radix-engine/src/system/kernel_modules/costing/fee_reserve.rs

```diff
@@ -260,6 +262,7 @@ impl SystemLoanFeeReserve {
         }
     }
 
+    #[trace_resources]
```

Comment:
Can we remove tracing on individual kernel modules, and add tracing for all methods in `KernelModuleMixer`?